src/Cubie/Corners.hs

```diff
@@ -0,0 +1,20 @@
+module Cubie.Corners where
+
+import Prelude hiding (length)
```

Comment:
Is Prelude something that is imported automatically/implicitly?

Reply:
Prelude is imported implicitly.
I am importing Prelude here to hide the length function, because Vector has it's own length function that I wanted instead. This is kind of left over from when I was using types other than Integer and I had my own length function in Cubie.Misc.

Reply:
The linter is complaining that this line is not necessary since vector is imported qualified